com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -463,6 +488,148 @@ public override void OnInspectorGUI()
             m_SerializedCamera.serializedObject.ApplyModifiedProperties();
         }
 
+        private void UpdateStackCemerasToOverlay()
+        {
+            int cameraCount = m_SerializedCamera.cameras.arraySize;
+            for (int i = 0; i < cameraCount; ++i)
+            {
+                SerializedProperty cameraProperty = m_SerializedCamera.cameras.GetArrayElementAtIndex(i);
+
+                var camera = cameraProperty.objectReferenceValue as Camera;
+                if (camera == null)
+                    continue;
+
+                var additionalCameraData = camera.GetComponent<UniversalAdditionalCameraData>();
+                if (additionalCameraData == null)
+                    continue;
+
+                Undo.RecordObject(camera, "Inspector Overlay Camera");
+                if (additionalCameraData.renderType == CameraRenderType.Base)
+                {
+                    additionalCameraData.renderType = CameraRenderType.Overlay;
+                    EditorUtility.SetDirty(camera);
+                }
+            }
+        }
+
+        private void UpdateStackCamerasOutput()
+        {
+            int cameraCount = m_SerializedCamera.cameras.arraySize;
+            for (int i = 0; i < cameraCount; ++i)
+            {
+                SerializedProperty cameraProperty = m_SerializedCamera.cameras.GetArrayElementAtIndex(i);
+                Camera overlayCamera = cameraProperty.objectReferenceValue as Camera;
+                if (overlayCamera != null)
+                    UpdateStackCameraOutput(overlayCamera);
+            }
+        }
+
+        private void UpdateStackCameraOutput(Camera camera)
+        {
+            Undo.RecordObject(camera, "Inspector Overlay Camera");
```

Comment:
Undo stack must be localized

https://confluence.unity3d.com/display/LOC/Editor+Localization+Coding+Guidelines

Reply:
Fixed, I checked the urp/core code seems like we do not do this, maybe we need jira for that?

Reply:
Yeah, a JIRA would be nice